### PR TITLE
Expose the UpdateApiBaseUrls through client

### PR DESCRIPTION
**Pull request by niladrikar0306**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ version: "3"
 services:
   autorest-gen: 
     image: reg-hk.agodadev.io/library/autorest-gen:${IMAGE_VERSION}
+    # build: .
+    # volumes:
+    #   - /Users/nkar/workspaces/temp/autorest/output:/app/output
```
Comment (kumarRaj) [inline docker-compose.yml:12]: Can you remove these?

**Review by kumarRaj — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (niladrikar0306) [reply docker-compose.yml:12]: It is helpful for local testing.

Comment (niladrikar0306) [reply docker-compose.yml:12]: If you want I can put some more comments to make it more readable.

Comment (niladrikar0306) [reply docker-compose.yml:12]: done. Added more comment as suggested.

Comment (kumarRaj) [reply docker-compose.yml:12]: Yes please add some comments in there

**Review by kumarRaj — APPROVED** [review]

**Pull request merged**